### PR TITLE
D8 nid 1619

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: .platform/solr_config/solrcore.properties [hunk 1]
```diff
@@ -1,16 +1,14 @@
 solr.replication.master=false
 solr.replication.slave=false
 solr.replication.pollInterval=00:00:60
-solr.replication.masterUrl=http://solr.internal:8080/solr
+solr.replication.masterUrl=http://localhost:8983/solr
```
Comment (johangant) [inline .platform/solr_config/solrcore.properties:4]: Is this right? Platform.sh would usually use port 8080 (hostname might vary with the service name change?) but this one looks like a Lando service which typically runs on port 8983

**Review by johangant — CHANGES_REQUESTED** [review]
LGTM, just the one possible red flag around the service hostname/port defined in the .platform config directory

File: .platform/solr_config/solrcore.properties (hunk 1 above)
Comment (omahm) [reply .platform/solr_config/solrcore.properties:4]: This is identical to the comment I left on Martin's PR and this morning on standup he said that 8983 oddly enough works.

Comment (johangant) [reply .platform/solr_config/solrcore.properties:4]: Okay, great. If it works it works :)

**Review by johangant — APPROVED** [review]

**Pull request merged**